docs/admin/authorization/rbac.md

```diff
@@ -78,42 +86,44 @@ metadata:
   name: read-pods
   namespace: default
 subjects:
-  - kind: User # May be "User", "Group" or "ServiceAccount"
-    name: jane
+- kind: User
+  name: jane
+  apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: Role
   name: pod-reader
   apiGroup: rbac.authorization.k8s.io
 ```
 
-`RoleBindings` may also refer to a `ClusterRole`. However, a `RoleBinding` that
-refers to a `ClusterRole` only applies in the `RoleBinding`'s namespace, not at
-the cluster level. This allows admins to define a set of common roles for the
-entire cluster, then reuse them in multiple namespaces.
+A `RoleBinding` may also reference a `ClusterRole` to grant the permissions to namespaced
+resources defined in the `ClusterRole` within the `RoleBinding`'s namespace.
+This allows administrators to define a set of common roles for the entire cluster,
+then reuse them within multiple namespaces.
 
 For instance, even though the following `RoleBinding` refers to a `ClusterRole`,
 "dave" (the subject) will only be able read secrets in the "development"
-namespace, the namespace of the `RoleBinding`.
+namespace (the namespace of the `RoleBinding`).
 
 ```yaml
 # This role binding allows "dave" to read secrets in the namespace "development"
```

Comment:
- [ ] Since this sentence starts with a capital letter, consider punctuating it too:
> in the namespace "development."